### PR TITLE
Updating contents for "KintoBlocks"

### PR DESCRIPTION
**Pull request by ajesh-cc**
Hi Noah,

Updated docs for adding "Build Arguments", "Helm Arguments" and moved contents of "Website" to "Manage-blocks" page under "docs/features/building" folder.

Also updated screenshot  in "docs/features/building/manage-blocks.md"
Please review and approve the changes.

Regards,
Ajesh

### CODE REVIEW

File: docs/features/building/helm-arguments.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Helm Arguments
+---
+
+## What are Helm Arguments?
+
+**Helm Arguments** are used to reconfigure the default configurations in **Helm Charts**. They are similar to environment variables. Each **branch** of a Custom Service KintoBlock will have their own Helm Arguments. It is also posisble to add multiple Helm Arguments to a Custom Service KintoBlock.
```
Comment (bappr) [inline docs/features/building/helm-arguments.md]: "they are similar to environment variables" -> "They will be used during the deployment".
"Each **branch** of a Custom Service KintoBlock will have their own Helm Arguments." -> wrong, it's one for all.
"posisble" -> typo

File: docs/features/building/helm-arguments.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The Helm Arguments value can be added on the **KintoBlock Manage** page or via **Configure Dependencies** button in Deployment Manage Page. It is recommended to assign a `Value` for each Helm Argument that is added to the Custom Service KintoBlock.
```
Comment (bappr) [inline docs/features/building/helm-arguments.md]: "It is recommended to assign a `Value` for each Helm Argument that is added to the Custom Service KintoBlock. "

Not recommended, you might want to leave it empty.
It's not env variables.

File: docs/features/building/helm-arguments.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+The Helm Arguments value can be added on the **KintoBlock Manage** page or via **Configure Dependencies** button in Deployment Manage Page. It is recommended to assign a `Value` for each Helm Argument that is added to the Custom Service KintoBlock. 
+
+
+## How to define Helm Arguments in a Custom Service KintoBlock?
+
+The Helm Arguments can be added to a Custom Service KintoBlock via the **KintoBlock Manage** page by following the steps mentioned below:
+
+1. Open a **Custom Service** KintoBlock.
+
+2. Select the required branch in your KintoBlock.
```
Comment (bappr) [inline docs/features/building/helm-arguments.md]: No need

File: docs/features/building/helm-arguments.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+3. Scroll down to the **Helm Arguments** section at the bottom of the form.
```
Comment (bappr) [inline docs/features/building/helm-arguments.md]: No, you have to click on "Settings"

File: docs/features/building/helm-arguments.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## Manage Helm Arguments
+
+It is possible to edit and delete a Helm Argument in Custom Service KintoBlock at any required point. The edit and delete operations are done in the **KintoBlocks Manage** page:
+
+1. Open a **Custom Service** KintoBlock.
+
+2. Select the required branch in your KintoBlock.
+
+3. Scroll down to the **Helm Arguments** section at the bottom of the form.
```
Comment (bappr) [inline docs/features/building/helm-arguments.md]: Same, go to settings

File: docs/features/building/manage-blocks.md [hunk 6]
```diff
@@ -4,24 +4,93 @@ title: Manage KintoBlocks
...
+
+4. Click on the **Save Changes** button.
+
+[Click Here](/docs/features/deploying/environment-variables) to Know more about **Environment Variables**.
+
+>Note: Above mentioned steps can be followed for **Microservice** and **Custom Service** for editing the properties like `KintoBlock Name`, `Language`, `Version`, `Description`, `Environment Variables` and `Hardware Configuration`.
+
+## Advanced Options for KintoBlock
+
+Advanced Options is used modify the KintoBlock with properties like `Protocol`, `Port`, `Build command`, `Start command` and `Doc` format are auto-filled based on the chosen language. These values are editable and can be changed if required.
```
Comment (bappr) [inline docs/features/building/manage-blocks.md]: "used modify" -> "used to modify"

Remove Doc, we disabled it

File: docs/features/building/manage-blocks.md [hunk 7]
```diff
@@ -4,24 +4,93 @@ title: Manage KintoBlocks
...
+
+>Note: For **Microservice KintoBlock** properties like `Protocol`, `Port`, `Build command`, `Start command` and `Doc` are availabe. 
+
+To edit Advanced Option, follow the steps mentioned below:
+
+1. Open a KintoBlock.
+
+2. Scroll down to **Builds & Repository** section.
+
+3. Expand **Advanced Options** to view the `Protocol`, `Port`, `Build command`, `Start command`, and `Doc` fields.
```
Comment (bappr) [inline docs/features/building/manage-blocks.md]: Remove doc

File: docs/features/building/manage-blocks.md (hunk 6 above)
Comment (ajesh-cc) [reply docs/features/building/manage-blocks.md]: Done

File: docs/features/building/manage-blocks.md (hunk 7 above)
Comment (ajesh-cc) [reply docs/features/building/manage-blocks.md]: Done.

File: docs/features/building/helm-arguments.md (hunk 1 above)
Comment (ajesh-cc) [reply docs/features/building/helm-arguments.md]: Done

File: docs/features/building/helm-arguments.md (hunk 2 above)
Comment (ajesh-cc) [reply docs/features/building/helm-arguments.md]: Done.

File: docs/features/building/helm-arguments.md (hunk 3 above)
Comment (ajesh-cc) [reply docs/features/building/helm-arguments.md]: Done.

File: docs/features/building/helm-arguments.md (hunk 4 above)
Comment (ajesh-cc) [reply docs/features/building/helm-arguments.md]: Done.

File: docs/features/building/helm-arguments.md (hunk 5 above)
Comment (ajesh-cc) [reply docs/features/building/helm-arguments.md]: Done.

**Pull request merged**